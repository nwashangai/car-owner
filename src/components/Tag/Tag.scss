@import 'sass-rem';
@import 'assets/variables';

.tag {
  background-color: map-get($grey, 200);
  margin: 0 rem(4px);
  padding: rem(7px);
  border-radius: rem(13px);
}
