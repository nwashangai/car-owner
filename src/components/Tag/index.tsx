// react libraries
import * as React from 'react';

// Styles
import './Tag.scss';

export default ({ tag }: any) => (
  <div className="tag">
    <span>{tag}</span>
  </div>
);
