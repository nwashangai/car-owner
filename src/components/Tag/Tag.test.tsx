// react libraries
import * as React from 'react';

// third-party libraries
import { mount } from 'enzyme';

// components
import Tag from '.';

describe('The Tag component', () => {
  it('displays Tag', () => {
    const wrapper = mount(<Tag color="red" />);

    const child = wrapper.find('.tag');
    expect(child).toMatchSnapshot();
    expect(child.length).toBe(1);
  });
});
