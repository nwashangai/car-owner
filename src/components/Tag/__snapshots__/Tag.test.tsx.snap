// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Color component displays color 1`] = `ReactWrapper {}`;

exports[`The Tag component displays Tag 1`] = `ReactWrapper {}`;
